astropy/cosmology/io/html.py

```diff
@@ -0,0 +1,131 @@
+import astropy.cosmology.units as cu
+import astropy.units as u
+from astropy.cosmology import Parameter, Planck18
+from astropy.cosmology.connect import readwrite_registry
+from astropy.cosmology.core import Cosmology
+from astropy.table import QTable
+
+from .table import from_table, to_table
+
+# format look-up for conversion, {original_name:new_name}
+format_table = {}

```

Comment:
It would be great to get a couple defaults filled in here, particularly for writing some tests.